parsec/core/gui/workspace_button.py

```diff
@@ -52,7 +59,7 @@ def __init__(
         self.switch_button.clicked.connect(self._on_switch_clicked)
 
         self.reencrypting = None
```

Comment:
Still usefull ?

Reply:
Yes, `reencrypting` is different from `reencryption_needs`, it is there to signal whether the workspace is currently reencrypting.